src/setup/rgw_service

```diff
@@ -39,7 +39,7 @@ done
 
 function start_rgw {
   machine_id=$(cat /etc/machine-id)
-  exec sudo /usr/bin/radosgw -f --name client.rgw-"$index" -c "$rgw_config_file" --no-mon-config &> "$log_file" &
+  exec /usr/bin/radosgw -f --name client.rgw-"$index" -c "$rgw_config_file" --no-mon-config &> "$log_file" &
```

Comment:
we can remove 'sudo' from 'create admin user' cmd also. (rgw.py file line no. 274 )